### PR TITLE
Publish to Sonatype

### LINKED ISSUE #24: Packages missing from any repositroy
Hey, plugin is gone today, you can't find it in any repository.

### PR DESCRIPTION
**Pull request by julienrf**
Also change organization name to `com.github.sbt`

Relates to #23, fixes #24

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -79,12 +79,14 @@ scriptedDependencies := {
...
-bintrayOrganization := Some("sbt")
-bintrayRepository := "sbt-plugin-releases"
-bintrayPackage := "sbt-javaagent"
-bintrayReleaseOnPublish := false
+publishMavenStyle := true
 licenses += "Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.html")
+scmInfo := Some(ScmInfo(url("https://github.com/sbt/sbt-javaagent"), "scm:git:git@github.com:sbt/sbt-javaagent.git"))
+homepage := scmInfo.value.map(_.browseUrl)
+developers := List(
+  Developer("pvlugter", "Peter Vlugter", "peter.vlugter@lightbend.com", url("https://github.com/pvlugter"))
```
Comment (pvlugter) [inline build.sbt:87]: Don't know if we have a general Lightbend team that we can use here? @ennru?

**Review by pvlugter — APPROVED** [review]
LGTM. Thanks!

**Comment by julienrf** [comment]
Thank you for the review! Before you merge, you will have to set the secrets PGP_PASSPHRASE and PGP_SECRET as explained here: https://github.com/sbt/sbt-ci-release#secrets

**Comment by julienrf** [comment]
Hey, would you have some time available soon to set the secrets and try publishing a milestone release?

File: build.sbt (hunk 1 above)
Comment (ennru) [reply build.sbt:87]:
```suggestion
  Developer("contributors", "Contributors", "https://github.com/sbt/sbt-javaagent/discussions", url("https://github.com/sbt/sbt-javaagent/graphs/contributors"))
```

**Comment by ennru** [comment]
> Hey, would you have some time available soon to set the secrets and try publishing a milestone release?

I do not have access to `com.github.sbt` at Sonatype.

**Comment by eed3si9n** [comment]
I think I've set up the Sonatype creds a while back, but it's missing GPG signing key. Let me create that too.

**Comment by eed3si9n** [comment]
Added a GPG signing key.

**Comment by julienrf** [comment]
Thank you Enno and Eugene for your help!

**Comment by julienrf** [comment]
I am going ahead, merging and trying to release an M1 version to see if the workflow works.

**Comment by pvlugter** [comment]
@julienrf great, thanks!

**Pull request merged**